### ISSUE TITLE
Improve unused CSS reporting

### ISSUE THREAD

**Issue by arggh** [sveltejs/sapper#375]
Would be nice, if I could get information on the file or component, where the unused CSS is.

Currently, this is what we get:

```
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (148:2)
146:   }
147:
148:   .item img {
       ^
149:     width: 100%;
150:     height: 100%;
```

**Comment by Rich-Harris** [comment]
Try updating! This was fixed yesterday in #371

**Comment by arggh** [comment]
This was the fastest issue fix ever! Much better now, thanks!
